src/main/groovy/com/github/eerohele/SaxonXsltTask.groovy

```diff
@@ -789,11 +831,11 @@ class SaxonXsltTask extends DefaultTask {
     // and output.
     @Internal
     protected List<String> getCommonArguments() {
-        Map<String, String> commonOptions = this.options.findAll { name, value ->
-            !['input', 'output'].contains(name)
-        }.asImmutable()
-
-        (commonOptions.inject(this.defaultArguments) { arguments, entry ->
+        def fileArguments = [makeSingleHyphenArgument(Argument.MAPPING.stylesheet, stylesheetFile)]
```

Comment:
What issue does this change fix? Whatever it is, it should be in a different commit.

Reply:
as of the gradle 7 deprecation warnings many inputs have to be declared and maps containing file names will always use the absolute path. I sure had #29 in mind when extracting the files to individual properties and therefor the change was necessary.